modules/kibana/src/internalClusterTest/java/org/elasticsearch/kibana/KibanaThreadPoolIT.java

```diff
@@ -61,4 +97,108 @@ public void testKibanaThreadPool() {
             }
         });
     }
+
+    public void testBlockedThreadPoolsRejectUserRequests() throws Exception {
```

Comment:
Nit: is this the right place for this test? This one is not Kibana specific, right?

Reply:
yes, in a way this is a 'negative test case' for what KibanaThreadPool test is doing. 
previously it was part of a super class in a testframework, not sure if it is worthy at the moment to refactor it into its own test class.

Reply:
OK